sklearn/ensemble/_forest.py

```diff
@@ -1822,10 +1825,13 @@ class ExtraTreesClassifier(ForestClassifier):
            The default value of ``n_estimators`` changed from 10 to 100
            in 0.22.
 
-    criterion : {"gini", "entropy", "log_loss"}, default="gini"
+    criterion : {"gini", "entropy", "log_loss", "hellinger"}, default="gini"
         The function to measure the quality of a split. Supported criteria are
-        "gini" for the Gini impurity and "log_loss" and "entropy" both for the
-        Shannon information gain, see :ref:`tree_mathematical_formulation`.
+        "gini" for the Gini impurity
+        "log_loss" and "entropy" both for the Shannon information gain,
+            see :ref:`tree_mathematical_formulation`
```

Comment:
I think ```:ref:`tree_mathematical_formulation` ``` better be put in the end of the `Note` bellow as it redirect to the section for model aspects including all criteria.

Reply:
done